composer.json

```diff
@@ -16,7 +16,8 @@
         }
     ],
     "require": {
-        "php": ">=7.2.0"
+        "php": ">=7.2.0",
+        "vimeo/psalm": "4.x-dev"
```

Comment:
can we move this to require-dev ?

Reply:
This was moved. Please take another quick look.